app/controllers/submit.js

```diff
@@ -1315,6 +1388,25 @@ export default Controller.extend(Analytics, BasicsValidations, COIValidations, N
             this.set('selectedProvider', this.get('currentProvider'));
             this.set('providerChanged', false);
         },
+        saveAuthorAssertions() {
+            this.get('metrics')
+                .trackEvent({
+                    category: 'button',
+                    action: 'click',
+                    label: `${this.get('editMode') ? 'Edit' : 'Submit'} - Save and Continue Author Assertions Section`,
+                });
+            const model = this.get('model');
+            model.set('hasDataLinks', this.get('hasDataLinks'));
+            model.set('dataLinks', this.get('dataLinks'));
+            model.set('whyNoData', this.get('whyNoData'));
+            model.save().then(this._moveFromAuthorAssertions.bind(this))
+                .catch(this._failMoveFromAuthorAssertions.bind(this));
+        },
+        discardAuthorAssertions() {
+            this.set('hasDataLinks', this.get('model.hasDataLinks'));
+            this.set('dataLinks', this.get('model.dataLinks'));
+            this.set('whyNoData', this.get('model.whyNoData'));
```

Comment:
should the discard option be tracked as well? I don't remember if it was asked to be.

Reply:
I don't remember any GA requirement for that.